gremlin-go/driver/resultSet.go

```diff
@@ -50,25 +50,60 @@ type channelResultSet struct {
 	statusAttributes map[string]interface{}
 	closed           bool
 	err              error
-	mux              sync.Mutex
+	waitSignal       chan bool
+	channelMux       sync.Mutex
+	waitSignalMux    sync.Mutex
```

Comment:
It's an electrical engineering shorthand for multiplexer, which has nothing to do with mutex, but I always used it in firmware.... I will fix it since it makes no real sense.